pages/ecosystem.vue

```diff
@@ -44,11 +44,22 @@
               {{ tierName }}
             </bx-tab>
           </bx-tabs>
-          <bx-search
-            class="ecosystem__search"
-            placeholder="Search using keywords like algorithms, simulator, or machine learning"
-            @bx-search-input="searchOnMembers($event.detail.value)"
-          />
+          <div class="ecosystem__toolbar cds--row">
+            <div class="ecosystem__search cds--col-lg-14 cds--col-md-6">
+              <bx-search
+                placeholder="Search using keywords like algorithms, simulator, or machine learning"
+                @bx-search-input="searchOnMembers($event.detail.value)"
+              />
+            </div>
+            <bx-dropdown
```

Comment:
I have a suggestion, take it or leave it.

- I think by default, we should sort by `name`
  - we can set the initial value to `sortByValue` (which we'll default to `"name"`)
- I think we should add the `label-text` to say "Sort by"
- we can remove the `trigger-content`

```vue
<bx-dropdown
  class="ecosystem__sort-dropdown cds--col-lg-2 cds--col-md-2"
  label-text="Sort by"
  :value="sortByValue"
  @bx-dropdown-selected="setSortValue($event.detail.item.value)"
>
```

Reply:
Resolved in https://github.com/Qiskit/qiskit.org/pull/3286/commits/186ea075d560882cfac850ff1f0ef5633910d571